insights/core/plugins.py

```diff
@@ -276,6 +276,8 @@ def _visit_parsers(parent, visitor):
 
 def get_dependency_graph(component):
     graph = defaultdict(set)
+    if not COMPONENT_DEPENDENCIES[component]:
```

Comment:
I asked @csams about these changes, and he said they don't need to be ported.  In addition, he has a very large change that will soon be applied to master that will likely make these changes obsolete.